### PR TITLE
packaging(debian): replace php 8.0 by 8.1

### PR DESCRIPTION
**Pull request by kduret**
## Description

packaging(debian): replace php 8.0 by 8.1

**Fixes** MON-16524

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 21.10.x
- [ ] 22.04.x
- [ ] 22.10.x
- [x] 23.04.x (master)

### CODE REVIEW

File: .github/docker/Dockerfile.centreon-web-bullseye [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+RUN <<EOF
+
+apt update
+
+rm -f /tmp/debs-centreon/centreon-23.04*.deb /tmp/debs-centreon/centreon-central-23.04*.deb
+apt install -y /tmp/debs-centreon/centreon-*.deb
+
+rm -f /usr/share/centreon/.env.local.php
+echo "APP_DEBUG=true" >> /usr/share/centreon/.env
+echo "DEBUG_LEVEL=100" >> /usr/share/centreon/.env
```
Comment (jeremyjaouen) [inline .github/docker/Dockerfile.centreon-web-bullseye:55]: Debug mode is enable by default ?

**Review by abuathier — APPROVED** [review]

**Review by wtermellil — APPROVED** [review]

File: .github/docker/Dockerfile.centreon-web-bullseye (hunk 1 above)
Comment (kduret) [reply .github/docker/Dockerfile.centreon-web-bullseye:55]: yes, in order to have error messages when a E2E test fails

File: .github/docker/Dockerfile.centreon-web-bullseye [hunk 2]
```diff
@@ -0,0 +1,130 @@
+FROM centreon.jfrog.io/docker/debian:bullseye AS web_fresh
+
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN <<EOF
```
Comment (abuathier) [inline .github/docker/Dockerfile.centreon-web-bullseye:5]: Le Heredoc Dockerfile, ca a l'air d'être une spécificité "récente" de BuildKit.
Est-ce bien supporté partout ? buildx etc ...

Comment (kduret) [reply .github/docker/Dockerfile.centreon-web-bullseye:5]: Yes, we always use docker with buildkit in the github workflows

**Review by paloth — APPROVED** [review]

**Pull request merged**